pkgs/development/python-modules/3to2/default.nix

```diff
@@ -0,0 +1,31 @@
+{ lib
+, buildPythonPackage
+, fetchurl
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "3to2";
+  version = "1.1.1";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
```

Comment:
ah, no, they published a zip instead of a tar.gz. I still need to fix fetchPypi to try both since on PyPI you're allowed to upload either, but not both.